### PR TITLE
Feature#11 search view

### LINKED ISSUE #11: Search view

### PR DESCRIPTION
**Pull request by MauricioNC**
# Description
Was create the view for the search results.

For this issue, just the front-end was implemented.
  
  ## Screenshots 
image

### CODE REVIEW

**Review by LuisGomez89 — APPROVED** [review]

**Review by JuanequeX — APPROVED** [review]
LGTM

File: EasyLearn/app/helpers/search_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module SearchHelper
+end
```
Comment (YilKanda) [inline EasyLearn/app/helpers/search_helper.rb]: If you don't use this file you can delete it.

File: EasyLearn/config/routes.rb [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
   get '/preferences', to: "preferences#index"
   get '/home', to: "home#index"
   get "/player", to: "player#index"
   get '/settings', to: "settings#index"
   
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
+  
   # Defines the root path route ("/")
   # root "articles#index"
```
Comment (YilKanda) [inline EasyLearn/config/routes.rb]: Remove this commented code.

**Review by BrandonVG — APPROVED** [review]

File: EasyLearn/app/helpers/search_helper.rb (hunk 1 above)
Comment (MauricioNC) [reply EasyLearn/app/helpers/search_helper.rb]: Done.

File: EasyLearn/config/routes.rb (hunk 2 above)
Comment (MauricioNC) [reply EasyLearn/config/routes.rb]: Done.

**Review by VictorElizalde — APPROVED** [review]
LGTM, nice css class without using the important! flag

**Pull request merged**